src/prognostic_equations/implicit/implicit_tendency.jl

```diff
@@ -174,16 +116,6 @@ function implicit_vertical_advection_tendency!(Yₜ, Y, p, t, colidx)
             ᶠgradᵥ(ᶜp[colidx] - ᶜp_ref[colidx]) +
             ᶠinterp(Y.c.ρ[colidx] - ᶜρ_ref[colidx]) * ᶠgradᵥ_ᶜΦ[colidx]
         ) / ᶠinterp(Y.c.ρ[colidx])
-    if turbconv_model isa EDMFX
-        for j in 1:n
-            @. Yₜ.f.sgsʲs.:($$j).u₃[colidx] =
-                -(
```

Comment:
The implicit code was setting this tendency. Now I'm only augmenting it in the explicit code. cc @szy21

Reply:
Ah, good to know. I will change the grid-scale u_3 tendency to += after this is merged.